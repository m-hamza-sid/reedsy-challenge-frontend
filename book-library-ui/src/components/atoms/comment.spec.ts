import { mount } from "@vue/test-utils";
import { describe, expect, test } from "vitest";
import Comment from "./comment.vue";

describe("comment", () => {
  test("should render with content prop", () => {
    const iContent = "some content";
    const wrapper = mount(Comment, { props: { content: iContent } });
    expect(wrapper.text()).toContain(iContent);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
