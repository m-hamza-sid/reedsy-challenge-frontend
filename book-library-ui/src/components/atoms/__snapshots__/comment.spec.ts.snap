// Vitest Snapshot v1

exports[`author > should render with name prop 1`] = `"<div class=\\"text-left\\">some name</div>"`;

exports[`comment > should render with content prop 1`] = `"<p class=\\"text-left\\">some content</p>"`;
